ai_genomics/analysis/tag_evaluation/add_oa_tag_thresholds.py

```diff
@@ -0,0 +1,73 @@
+"""Script to add OA tag thresholds in labelled tag extraction validation dataset."""
+import ast
+import pandas as pd
+
+from ai_genomics.getters.data_getters import load_s3_data, save_to_s3
+from ai_genomics import bucket_name
+from ai_genomics.analysis.tag_evaluation.make_tag_validation_dataset import (
+    load_sample_tags,
+)
+
+LABELLED_TAG_EVALUATION_DATASET = load_s3_data(
+    bucket_name,
+    "inputs/ai_genomics_samples/labelled_ai_genomics_sample_validation_set.csv",
+)
+
+
+def get_oa_tags_above_threshold(
+    oa_samples: pd.DataFrame, threshold: int
+) -> pd.DataFrame:
+    """Generates column of OpenAlex tags above a threshold."""
+    oa_samples["scores"] = oa_samples["scores"].apply(
+        lambda x: [i for i, score in enumerate(x) if score > threshold]
```

Comment:
Minor point, but it's best to specify if a threshold is a min or max threshold.

In either case, I would use `<=` or `>=` as `min_threshold` implies 'minimum allowed value'. To be double safe then specify in the docstring, e.g.

```
min_threshold: Minimum allowed value. Concepts must have a score
    greater than or equal to this to be included.
```